test/not-automated/multiprocessing-threading/multithreading.py

```diff
@@ -0,0 +1,38 @@
+from threading import Thread
+import os
+
+def threaded_function():
+    top = "/home/nara/mntblobfuse/128/wiki_pretrain/"
```

Comment:
We want to make this script generic and add it to the pipeline ? I see the paths are hard-coded here.